modules/ipi-modify-a-disconnected-registry-config-yaml.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// install/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-modify-a-disconnected-registry-config-yaml_{context}"]
+
+= Modify the `install-config.yaml` file to use the disconnected registry (optional)
+
+On the provisioner node, the `install-config.yaml` file should use the newly created pull-secret from the `pull-secret-update.txt` file. The `install-config.yaml` file must also contain the disconnected registry node's certificate and registry information.
+
+.Procedure
+
+. Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces:
++
+[source,terminal]
+----
+$ echo "additionalTrustBundle: |" >> install-config.yaml
+$ sed -e 's/^/  /' /opt/registry/certs/domain.crt >> install-config.yaml
+----
+
+. Add the mirror information for the registry to the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ echo "imageContentSources:" >> install-config.yaml
+$ echo "- mirrors:" >> install-config.yaml
+$ echo "  - registry.example.com:5000/ocp4/openshift4" >> install-config.yaml
+$ echo "  source: quay.io/openshift-release-dev/ocp-release" >> install-config.yaml
+$ echo "- mirrors:" >> install-config.yaml
+$ echo "  - registry.example.com:5000/ocp4/openshift4" >> install-config.yaml
+$ echo "  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev" >> install-config.yaml
```

Comment:
Those `echo` commands looks not quite elegant, maybe we can just reference the steps in  
https://docs.openshift.com/container-platform/4.10/installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.html#installation-initializing_installing-restricted-networks-aws-installer-provisioned
about how to editing the install-config.yaml file?